04.ls/ls.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+COL_MAX = 3
+PADDING = 2
+
+def main
+  path = ARGV[0] || '.'
+  file_names = Dir.children(path).sort if FileTest.directory?(path)
+  file_names or return
```

Comment:
```suggestion
  FileTest.directory?(path) or return
  file_names = Dir.children(path).sort
```

みたいな感じにして、file_namesの代入を実行する前に処理を中断する方がいいかなと思いました